### PR TITLE
add multinote metrics

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -56,12 +60,14 @@ An event fired when the user actively navigates away from the Notes sidebar. Inc
...
 - `cd10`
+- `cd11`
 
 #### `changed`
-An event fired when the user completes a change of the content of the notepad. It prospectively begins when a user focuses on the notepad's editable area, and ends when the user either 1) closes the sidebar, or 2) does not make any changes in 20 seconds. Includes:
+An event fired when the user completes a change of the content of the active notepad. It prospectively begins when a user focuses on the notepad's editable area, and ends when the user either 1) closes the sidebar, or 2) does not make any changes in 20 seconds. Includes:
 
 - `ec` - `notes`
 - `ea` - `changed`
+- `ev` - `<integer index of active note>`
```
Comment (vladikoff) [inline docs/metrics.md]: "do people work on top notes", need to see what to do about `index`, because we sort.

File: docs/metrics.md [hunk 2]
```diff
@@ -88,6 +96,38 @@ An event fired when the user tries to drag or drop a content into the notepad.
...
+
+#### `new-note`
+Event fired when a user creates a new note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `list-view` where `in-note` refers to the meatball menu flow and `list-view` refers to the button in the list view
+- `cd10`
+- `cd11`
+
+#### `export`
```
Comment (vladikoff) [inline docs/metrics.md]: https://github.com/mozilla/notes/blob/df3df4784bf0107a5db85ba40668b5a55db7489e/src/sidebar/app/actions.js#L159

Comment (johngruen) [reply docs/metrics.md]: I actually think we should change it...that way we can filter export type on the `el`

Comment (vladikoff) [reply docs/metrics.md]: great idea!

Comment (johngruen) [reply docs/metrics.md]: High five

**Comment by vladikoff** [comment]
@sebastienbarbier feel free to just use this branch for the code PR

File: docs/metrics.md [hunk 3]
```diff
@@ -88,6 +94,38 @@ An event fired when the user tries to drag or drop a content into the notepad.
 - `cd5`
 - `cd6`
 - `cd10`
+- `cd11`
+
+#### `new-note`
+Event fired when a user creates a new note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `list-view` where `in-note` refers to the meatball menu flow and `list-view` refers to the button in the list view
```
Comment (sebastienbarbier) [inline docs/metrics.md]: A third option to create a new note is using `send-to-note` while being on the `list-view`. I will add that 👍

**Comment by sebastienbarbier** [comment]
v4 does no longer trigger the metric `limit-reached` since limitation has been removed. Should I remove it from docs or should be keep tracking it ?

File: docs/metrics.md [hunk 4]
```diff
@@ -88,6 +94,38 @@ An event fired when the user tries to drag or drop a content into the notepad.
...
+- `ea` - `export`
+- `el` - one of `html`, `markdown`, `plaintext`, `richtext`
+- `cd10`
+- `cd11`
+
+### `delete-note`
+Event fired when a user deletes a note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `exit-blank-note` which occurs when the user navigates back from a new note without adding content
```
Comment (sebastienbarbier) [inline docs/metrics.md]: We no longer delete note on exit event, but on change event if content is empty (that solve many issues with multi window syncing). So I guess I will rename `exit-blank-note` to `blank-note`

**Comment by vladikoff** [comment]
@sebastienbarbier for metrics that we stop sending we just mark them '(deprecated)'. in the title, we have one like that already

**Comment by sebastienbarbier** [comment]
@vladikoff can you review for me please and let me know if I missed anything 👍

**Pull request merged**